content/docs/introduction/regions.md

```diff
@@ -6,6 +6,7 @@ redirectFrom:
   - /docs/conceptual-guides/regions
 updatedOn: '2024-01-11T14:49:37.565Z'
 ---
+
 Neon supports project deployment in several regions. We recommended that you select the region closest to your application server to reduce latency between your Neon project and your application.
```

Comment:
I'm sure it's fine as is, but would it be more relevant/accurate to say "Neon database" over "Neon project", since its the db you connect to?

Reply:
Makes sense! changed.